lib/levensthein-distance.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Gives you a array with filled with 0...amount.
+ * @param {number} amount
+ * @returns {number[]}
+ */
+function range(amount) {
+  const range = Array.from({length: amount});
```

Comment:
Just use `Array(amount)`. There is no advantage to use `Array.from` here.

1. You have to create a new object.
2. `Array.from` has to visit each index and copy the value (undefined) from the object to the array.

Reply:
The `unicorn/no-new-array` rule is enabled thus not allowing having `new Array(amount)`